digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/RoadAddressDAOSpec.scala

```diff
@@ -30,6 +32,46 @@ class RoadAddressDAOSpec extends FunSuite with Matchers {
     }
   }
 
+  test("insert roadaddress duplicate info check") {
```

Comment:
Would be better to use intercept to catch the exception and to check that error message

Reply:
You are right, its done